abc.templates/infra/terraform.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 1.0"
+
+  backend "gcs" {
+    bucket = "REPLACE_BUCKET_NAME"
+    prefix = "REPLACE_BUCKET_PREFIX"
+  }
+
+  required_providers {
+    google = {
+      version = ">= 4.45"
```

Comment:
I wonder if this should be a template input with a default value.

Reply:
I dont know if default values are supported yet